packages/clerk-js/src/ui/UserProfile/MfaBackupCodePage.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { useWizard, Wizard } from '../common';
+import { Button, Text } from '../customizables';
+import { withCardStateProvider } from '../elements';
+import { FormButtonContainer } from './FormButtons';
+import { MfaBackupCodeCreatePage } from './MfaBackupCodeCreatePage';
+import { NavigateToFlowStartButton } from './NavigateToFlowStartButton';
+import { ContentPage } from './Page';
+
+export const MfaBackupCodePage = withCardStateProvider(() => {
+  const wizard = useWizard();
+
+  return (
+    <Wizard {...wizard.props}>
+      <AddBackupCode
+        title={'Add multi-factor authentication'}
+        onContinue={wizard.nextStep}
+      />
+
+      <MfaBackupCodeCreatePage />
+    </Wizard>
+  );
+});
+
+type AddBackupCodeProps = {
+  title: string;
+  onContinue: () => void;
+};
+
+const AddBackupCode = (props: AddBackupCodeProps) => {
+  const { title, onContinue } = props;
+
+  return (
+    <ContentPage.Root headerTitle={title}>
+      <Text>Backup codes will be enabled for this account.</Text>
```

Comment:
❓ @nikosdouvlis Should we add all these strings in the new localization context or will we do this in a separate PR?

Reply:
Hello Sokrati, I've already talked with Haris about that. Lets "hardcode" them for now, and I'll do a final pass to localize all strings under `/UserProfile`